docs/source/contents/kubernetes/storage-secrets/index.md

```diff
@@ -0,0 +1,147 @@
+# Storage Secrets
+
+Inference artifacts referenced by Models can be stored in any of the storage backends supported by [Rclone](https://rclone.org/).
+This includes local filesystems, AWS S3, and Google Cloud Storage (GCS), among others.
+Configuration is provided out-of-the-box for public GCS buckets, which enables the use of Seldon-provided models like in the below example:
+
+```{literalinclude} ../../../../../samples/models/sklearn-iris-gs.yaml 
+:language: yaml
+```
+
+This configuration is provided by the Kubernetes `secret` `seldon-rclone-gs-public`.
+It is made available to `Servers` via a [centralised configmap](#central-config-map).
```

Comment:
This is in context of K8s? Instead of implying that one can control this via ConfigMap I believe we added recently option for just referencing default secrets?

Reply:
Oh yeah, it is not really customizable via Helm currently...

Reply:
I've updated the link and in the linked section discuss how to use this with `SeldonRuntime` resources, which I think is what you mean?  The "default" secrets are handled through a centralised configmap, but I don't think users need to be very aware of that detail

Reply:
Updated in https://github.com/SeldonIO/seldon-core/pull/4969/commits/e9082ef1fa60691ae32e35e0b31e0dfc617bdd10

Reply:
Ok, now I like this part more :+1: